components/Navbar.tsx

```diff
@@ -90,9 +90,9 @@ const HNavbar = ({ brand, links, actionButton }: NavbarProps) => {
               </StyledNavLink>
             ))}
           </Nav>
-          <div className="mt-lg-0 mt-3 mb-3 mb-lg-0">
+          {/* <div className="mt-lg-0 mt-3 mb-3 mb-lg-0">
```

Comment:
Without the button, the navbar links seem to be randomly aligned since they're not quite at the centre of the page. If we change from `mx-auto` to `ml-auto` for the class on the `Nav`, the links become right aligned which might look cleaner?

Reply:
TRUE yea that's a good point - let's do that

Reply:
alrighty should be fixed now 👍